api/src/main/java/org/codice/ddf/admin/api/validation/SourceValidationUtils.java

```diff
@@ -41,8 +53,7 @@
         return errors;
     }
 
-    public static List<ConfigurationMessage> validateCswFactoryPid(String factoryPid,
-            String configId) {
+    public List<ConfigurationMessage> validateCswFactoryPid(String factoryPid, String configId) {
```

Comment:
We addressed this in my PR replaced it with a generic method that takes the list of PIDs you want to validate against.